wye/templates/profile/college_poc_dashboard.html

```diff
@@ -0,0 +1,157 @@
+                            </tr>
+                        </thead>
+                        <tbody>
+                            {% for workshop in workshop_requested_under_poc %}
+                                <tr>
+                                    <td>{{ workshop.requester }}</td>
+                                    <td>{{ workshop.get_presenter_list|join:", " }}</td>
+                                    <td>{{ workshop.expected_date }}</td>
+                                    <td>{{ workshop.location }}</td>
+                                    <td>{{ workshop.workshop_section }}</td>
+                                </tr>
+                            {% endfor %}
+                        </tbody>
+                    </table>
+                </div>
+            {% endif %}
+        </div>
+    </div>
+</div>
+
+<div class="push-4-bottom">
+    <div class="row">
+        <div class="col-xs-6">
+            <h3 class="space-1q-top">Workshops Accepted</h3>
+        </div>
+    </div>
+    <div class="row push-1">
+        <div class="col-sm-12">
+            {% if not workshops_accepted_under_poc %}
+                <p class="gray sub"><em>There are no workshops that have been accepted.</em></p>
+            {% else %}
+                <div class="table-responsive">
+                    <table class="table table-bordered sub">
+                        <thead>
+                            <tr>
+                                <th>Organization</th>
+                                <th>Presenter</th>
+                                <th>Date</th>
+                                <th>Location</th>
+                                <th>Section</th>
+                            </tr>
+                        </thead>
+                        <tbody>
+                            {% for workshop in workshops_accepted_under_poc %}
+                                <tr>
+                                    <td>{{ workshop.requester }}</td>
+                                    <td>{{ workshop.get_presenter_list|join:", " }}</td>
+                                    <td>{{ workshop.expected_date }}</td>
+                                    <td>{{ workshop.location }}</td>
+                                    <td>{{ workshop.workshop_section }}</td>
+                                </tr>
+                            {% endfor %}
+                        </tbody>
+                    </table>
+                </div>
+            {% endif %}
+        </div>
+    </div>
+</div>
+
+<div class="push-4-bottom">
+    <div class="row">
+        <div class="col-xs-6">
+            <h3 class="space-1q-top">Workshops Completed</h3>
+        </div>
+    </div>
+    <div class="row push-1">
+        <div class="col-sm-12">
+            {% if not workshop_completed_under_poc %}
+                <p class="gray sub"><em>There are no workshops that are completed.</em></p>
+            {% else %}
+                <div class="table-responsive">
```

Comment:
This rendering is repeated across multiple templates. Can you use `include` or inclusion tags ? https://docs.djangoproject.com/en/1.8/howto/custom-template-tags/#inclusion-tags